.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -3,21 +3,31 @@ name: New release template
 about: Template for creating new releases of Durable Functions
 title: ''
 labels: ''
-assignees: cgillum
+assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
```

Comment:
I may want to include the rotation order, but we haven't actually settled on one yet.

I'm fine randomizing it, or we could let people sign up for what they are interested in for the first release and just rotate from there.